AngouriMath/Functions/Evaluation/Evaluation.Discrete.Classes.cs

```diff
@@ -335,5 +335,24 @@ protected override Entity InnerSimplify()
                 }
             }
         }
+
+        partial record Phif
+        {               
+            private Entity InnerCompute(Entity entity)
+            {
```

Comment:
Fun fact: it could be written like this
```cs
private Entity InnerCompute(Entity entity) => (entity as Integer)?.Phi() ?? this;
```
But let's keep it more readable. So just remove the unnecessary curly braces, and it will be ok

Reply:
Once it's done & tests finish running, I merge